examples/04_glm_first_level/plot_spm_multimodal_faces.py

```diff
@@ -63,110 +55,124 @@
 # Let's create mean image for display purposes.
 mean_image = mean_img(fmri_img, copy_header=True)
 
+
 # %%
-# Make the design matrices.
-import numpy as np
-import pandas as pd
+# We can confirm there are only 2 conditions in the dataset.
+#
+
+# %%
+# Fit the :term:`GLM` for the 2 runs
+# by specifying a FirstLevelModel and then fitting it.
+# %%
+# Specify timing and design matrix parameters.
+
+# repetition time, in seconds
+t_r = 2.0
+# Sample at the beginning of each acquisition.
+slice_time_ref = 0.0
```

Comment:
I think that for datasets in general, this type of info should be part of the 'metadata' returned by the fetcher, rather than be hardcoded in examples.